docs/dashboards/reports-and-components.md

```diff
@@ -7,20 +7,24 @@ The **Report Menu** is where the true power of Sematext's Dashboards lies. This
 
 Here's how you can **correlate Metrics with Events and Logs**, and view it all in a single view.
 
-![Dashboard Reports](../images/dashboards/reports.png)
+![Dashboard Reports](../images/dashboards/elasticsearch-dashboard.png)
 
 ## Dashboard Components
 
 The image below shows a common Monitoring App component showing its functionality and interactive elements. Using the component settings dropdown menu, you can easily add any metrics components from one or more Apps to the Dashboard. Create correlations reports, observe and analyze how multiple technologies coexist in your system. Zoom and time shift through charts, display a single component in full view, create alerts, and effortlessly share reports across your organization. 
 
-![Metrics Dashboards Component](https://sematext.com/docs/images/guide/dashboards/metrics-dashboard-component.png "Metrics Dashboards Component")
+![Metrics Dashboards Component](../images/dashboards/metrics-dashboard-component-2.png)
 
-Just like a Monitoring App component, a Logs App components have interactive elements as well. Here you can choose from Log Tables, Logs counts, and much more. Use the **+ Add New Component** button to add a Logs App component to your Dashboard, and combine them with Monitoring App components for full observability of your stack. 
-
-![Logs Dashboards Component](https://sematext.com/docs/images/guide/dashboards/logs-dashboard-component.png "Logs Dashboards Component")
+Just like a Monitoring App component, a Logs App components have interactive elements as well. Here you can choose from Log Tables, Logs counts, and much more. Use the **+ Add New Component** button to add a Logs App component to your Dashboard, and combine them with Monitoring App components for full observability of your stack. Refer to [Logs Component Settings](https://sematext.com/docs/logs/reports-and-components/#logs-component-settings) for detailed information.
```

Comment:
What do we mean by "interactive elements"?

Reply:
updated.